pkg/cli/mirror/mirror.go

```diff
@@ -504,3 +516,22 @@ func (o *MirrorOptions) generateAllICSPs(mapping image.TypedImageMapping, dir st
 
 	return WriteICSPs(dir, allICSPs)
 }
+
+func (o *MirrorOptions) checkErr(err error, acceptableErr func(error) bool) error {
+	if err != nil {
```

Comment:
Fail fast:
```golang
if err == nil {
    return 
}
```
is preferred over what you have here.

Reply:
Great point. Updated.